docs/private-networks/how-to/configure/consensus/clique.md

```diff
@@ -164,6 +168,159 @@ At each epoch transition, Clique discards all pending votes collected from recei
 
 Define the number of blocks between epoch transitions in the [Clique genesis file](#genesis-file).
 
+## Transitions
+
+The `transitions` genesis configuration item allows you to specify a future block number at which to
+change the Clique network configuration in an existing network.
+For example, you can update the [block time](#configure-block-time-on-an-existing-network) and
+whether to [create empty blocks](#configure-empty-blocks-on-an-existing-network).
```

Comment:
```suggestion
choose whether to [create empty blocks](#configure-empty-blocks-on-an-existing-network).
```

Reply:
The meaning is "you can update the block time, and _you can update_ whether to create empty blocks..." so adding "choose" would change the meaning.